web/src/components/DLP/DandisetOwnersDialog.vue

```diff
@@ -100,6 +100,35 @@
       </v-col>
       <v-col class="grey lighten-3">
         <div class="my-6">
+          <v-dialog
+            v-model="selfRemovalWarningDialog"
+            width="40vw"
+          >
+            <v-card>
+              <v-card-title>
+                Remove yourself from this Dandiset?
+              </v-card-title>
+              <v-card-subtitle>
+                To regain ownership of this dandiset, you will
+                need another owner or an admin to add you.
+              </v-card-subtitle>
+              <v-card-actions>
+                <v-spacer />
+                <v-btn
+                  color="secondary"
```

Comment:
The rest of the `v-btn`'s on the DLP have their shadows removed, I think this would apply here for UI style consistency
```suggestion
                  color="secondary"
                  elevation="0"
```